providers/default.rb

```diff
@@ -54,6 +54,13 @@
     code 'npm -g install npm'
   end
 
+  # hacky fix to prevent filling up /tmp due to bug with npm
+  execute 'Clear temp files' do
+    command "find . -depth -iname 'npm-*' -type d -exec rm -r \"{}\" \\;"
```

Comment:
I don't feel good about this. Can you set it directly to ``find /tmp ...``. I know that cwd is set to it, but it'll make me feel better if the command actually does what you're wanting.